app/user/[id]/@user/default.tsx

```diff
@@ -1,14 +1,30 @@
+import { Avatar } from '@ost-cas-fee-adv-23-24/design-system-component-library-team-batman';
+import Image from 'next/image';
+
 import { GET_USER_BY_ID } from '@/utils/api/api-actions-user';
 import { APIError } from '@/utils/api/api-service-base';
 import { notFound } from 'next/navigation';
+import { ProfileImage } from './profileImage';
 
 export default async function Page({ params }: { params: { id: string } }) {
   try {
     const user = await GET_USER_BY_ID({ id: params.id });
     return (
-      <div className="overflow-auto">
-        <h1>USER PROFILE</h1>
-        <code>{JSON.stringify(user)}</code>
+      <div className="py-4 px-6 md:py-8 md:px-12 rounded-xl order-1 border-transparent lg:max-w-3xl">
+        <div className="relative">
+          <div className="mb-4 relative h-[320px] cursor-pointer overflow-hidden rounded-m bg-primary-600 object-contain">
+            <Image src={ProfileImage} alt="profile image" className="duration-200 ease-in-out hover:opacity-50" fill />
```

Comment:
du hast hier einige classnames verwendet die wir in unserer tailwidnconfig ausgeschlossen haben.  
Bitte achte darauf, dass die richtigen classnames verwendet werden und entferne die, die es nicht gibt.  

- `order-1` braucht es nicht, sonst landet es zu unterst, soll ja ganz oben sein.
- allgemein braucht es nicht zwei divs, der mit `relative` genügt